Library/Homebrew/cmd/update-report.rb

```diff
@@ -104,6 +105,35 @@ def update_report
       next unless tap.git?
       next if tap.core_tap? && ENV["HOMEBREW_INSTALL_FROM_API"].present? && args.preinstall?
 
+      # HOMEBREW_CORE_MERGE_MAINTAINER can be set by the maintainers
+      # who merge homebrew-core into linuxbrew-core to avoid being migrated
+      # We only migrate HOMEBREW_DEVELOPER users right now
+      # Don't migrate on CI either
+      if ENV["HOMEBREW_DEVELOPER"].present? &&
+         ENV["HOMEBREW_CORE_MERGE_MAINTAINER"].blank? &&
+         ENV["GITHUB_ACTIONS"].blank? &&
+         tap.core_tap? &&
+         tap.remote == "https://github.com/Homebrew/linuxbrew-core"
+        puts_stdout_or_stderr "Migrate from linuxbrew-core to homebrew-core" unless args.quiet?
+
+        LINUXBREW_CORE_MIGRATION_LIST.each do |name|
+          formula = Formula[name]
+          next unless formula.any_version_installed?
+
+          latest_version = formula.installed_prefixes.last.version
```

Comment:
`formula.installed_prefixes.last` is a `PathName` object, and `.version` does not give the expected thing. We need a different way to find the latest installed version.

Reply:
Try `installed_kegs` instead.

Reply:
Great. I think I got the right way of doing it now.